### ISSUE TITLE
Support English menus and labels

### ISSUE THREAD

**Issue by rafi** [ripperhe/Bob#41]
Amazing software! Really. Wonderful work to help people communicate with each-other from different languages of the world.

My problem currently is that I'm translating Bob's menus with Bob. Dog-fooding indeed :-)
I'm using Bob mainly to translate Chinese to English, it's really great, but it's hard to memorize the labels and menus.  Can I help in any way?

**Comment by ripperhe** [comment]
@rafi 
I can do internationalization for Bob, but my English is not very good. Maybe the text I translated will be weird. Can you fix it for me then?

Yes, these sentences are also translated by Bob...

**Comment by rafi** [comment]
@ripperhe Yes, I'm willing to help with translating English with you on a branch. Just let me know when to start :)

**Comment by ripperhe** [comment]
@rafi Ok, I'll let you know when I start

**Comment by ww7** [comment]
Can we have English interface? How we can help with this?

**Comment by Flute7** [comment]
I am also happy to help whenever necessary!

**Comment by freeurmind** [comment]
A config option to switch between English UI and Chinese UI is appreciated.
